### DISCUSSION TITLE
CSV import - date datatype

### DISCUSSION THREAD

**Discussion by Cinbert** [Budibase/budibase#1791] [Help] [answered]
I am a new user and trying to get my first app up and running. I created some test data in a csv file. I have tried a number of ways to format the date field including ISO8601 formating - 2020-09-24T13:12:04.622Z  but the table is created with no rows of data.   

There is clearly a failure somewhere but it is not being reported to me.  I only figured out that the import was failing on the date field through standard debugging steps.  My table rows only showed up after I specified the date field as a text data type.   

I have attached my small test data file.
[GIP_Tables.csv](https://github.com/Budibase/budibase/files/6690120/GIP_Tables.csv)

**Comment by mjashanks** [comment]
Hi @Cinbert. I can recreate this bug, thanks for that. The import is not handling blank dates, but it is supposed to.

I was able to import your CSV, by making sure that there are no blank values in the `EventDate` column. You could maybe set this to something like `1800-01-01T00:00:00.000Z`. I realise that this might not be suitable for you - we will get this fixed.

Otherwise, the formatting you are using for dates is correct.

**Reply by mjashanks** [reply]
Creating an issue from this, so that it is on the roadmap to be fixed: https://github.com/Budibase/budibase/issues/1794

**Comment by mike12345567** [comment, marked as answer]
This issue has been resolved as of the latest release.

**Reply by Cinbert** [reply]
Brilliant. Working now.  Thanks for the quick response.

**Comment by NicolasFabre81** [comment]
Hello,
I just checked that in my CSV import dates are parsed "in the english way" : MM/DD/YYYY
My dates are french : DD/MM/YYYY
How could I have them imported correct ? Is there any localisation parameter anywhere ?
